src/libcharon/plugins/kernel_super/system_events/empty_system_events/empty_system_events.h

```diff
@@ -0,0 +1,20 @@
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "../../libcharon/plugins/kernel_super/system_events/system_events.h"
```

Comment:
`../../libcharon/plugins/kernel_super/system_events/system_events.h` doesn't exist from `src/libcharon/plugins/kernel_super/system_events/empty_system_events/`. Luckily the makefiles auto include `src/libcharon` so this works.
But anyways this looks weird, either use absolute paths or relative one (as in `sl_system_events.h`). Whatever you choose make sure its the same between this file and `sl_system_events.h`